docs/source/examples.rst

```diff
@@ -68,7 +68,6 @@ Sparse Coding
    examples/sparsecode_admm
    examples/sparsecode_pgm
    examples/sparsecode_poisson_blkarr_pgm
```

Comment:
Since `sparsecode_poisson_pgm` has been deleted, do we still need the `blkarr` in the name of the remaining script, i.e. perhaps just rename `sparsecode_poisson_blkarr_pgm` to `sparsecode_poisson_pgm`?

Reply:
Done (8450359).

Reply:
Un-done to make Github diff less confused.

Reply:
But I think we really do want the renaming in this PR. Let's not drag this out. I'll approve the PR, if you'll re-rename and we can clean up anything that we don't see due to the renaming confusion post-merge.